systemtest/src/test/java/io/strimzi/systemtest/operators/ClusterOperatorRbacST.java

```diff
@@ -48,27 +49,28 @@ void testCRBDeletionErrorIsIgnoredWhenRackAwarenessIsNotEnabled(ExtensionContext
         String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
 
         // 060-Deployment
-        clusterOperator = new SetupClusterOperator.SetupClusterOperatorBuilder()
-            .withExtensionContext(extensionContext)
-            .withNamespace(Constants.INFRA_NAMESPACE)
+        this.clusterOperator = this.clusterOperator.defaultInstallation(extensionContext)
             .withClusterOperatorRBACType(ClusterOperatorRBACType.NAMESPACE)
+            .withWatchingNamespaces(Constants.TEST_SUITE_NAMESPACE)
+            .withBindingsNamespaces(Arrays.asList(Constants.CO_NAMESPACE, Constants.TEST_SUITE_NAMESPACE))
             .createInstallation()
             .runBundleInstallation();
 
+        cluster.setNamespace(Constants.TEST_SUITE_NAMESPACE);
+
         String coPodName = kubeClient().getClusterOperatorPodName(clusterOperator.getDeploymentNamespace());
         LOGGER.info("Deploying Kafka: {}, which should be deployed even the CRBs are not present", clusterName);
 
         resourceManager.createResourceWithWait(extensionContext, KafkaTemplates.kafkaEphemeral(clusterName, 3).build());
-
         LOGGER.info("CO log should contain some information about ignoring forbidden access to CRB for Kafka");
-        String log = cmdKubeClient().execInCurrentNamespace(Level.DEBUG, "logs", coPodName).out();
+        String log = cmdKubeClient().namespace(clusterOperator.getDeploymentNamespace()).execInCurrentNamespace(Level.DEBUG, "logs", coPodName).out();
```

Comment:
Yes, because we are grabbing logs from Cluster Operator. I can change it to `Constants.CO_NAMESPACE`.

Reply:
Yeah as I mentioned, I would maybe completely remove the `getDeploymentNamespace()` as it will be always the same. But it can be done in different PR as well.

Reply:
Yeah, make sense :)